security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -40,6 +40,10 @@ import (
 	"istio.io/pkg/log"
 )
 
+var (
+	cacheLog = log.RegisterScope("cacheLog", "cache debugging", 0)
```

Comment:
I think we normally (at least in pilot?) don't have the "log" suffix. So it would just be `cache`. a bit less noise for the logs that way

Reply:
I try to keep the naming consistent across all files. But in some files, e.g. vault/client.go, there is already a struct vaultClient, so I use vaultClientLog. We can come up with better and shorter names if this causes too much noise.

Reply:
I don't care about the variable name, just the name of the actual scope. Could be

`cacheLog = log.RegisterScope("cache", "cache debugging", 0)`

The log is unneeded and makes the logs bloated a little bit